must-gather/collection-scripts/gather

```diff
@@ -17,21 +20,36 @@ group_resources=()
 # Get namespace of node-maintenance-operator - where it is installed
 NMO_NAMESPACE=$(oc get subs -A --field-selector=metadata.name=node-maintenance-operator -o jsonpath='{.items[*].metadata.namespace}')
 
-# Get nmo logs - Nodes, CRD, and CRs
+# Get NMO Information - nodes' object, CRD, pod's logs, CRs, cluster's resource usage, and nodes' logs
+
+# Get Nodes' names
+NODES_NAMES=($(oc get nodes -o jsonpath='{.items[*].metadata.name}{"\n"}'))
 
-# Nodes
+# Nodes' objects
 group_resources+=(nodes)
 
-# NMO CRD
+# NMO's CRD
 NMO_CRD=$(oc get crds -o jsonpath='{range .items[*]}{"crd/"}{.metadata.name}{"\n"}{end}' | grep 'nodemaintenance.medik8s' | sed -z 's/\n/ /g')
 named_resources+=(${NMO_CRD})
 
-# node maintenance CRs
+# NMO's POD name
+NMO_POD=$(oc get pods -o jsonpath='{range .items[*]}{"pod/"}{.metadata.name}{"\n"}{end}' -n ${NMO_NAMESPACE}  | grep 'node-maintenance')
```

Comment:
I think you need to deal with multiple pods, e.g. when the node NMO is running on is drained, you might see the old draining pod, and a new pod.

Reply:
Good point. Now the container can gather logs from all NMO running pods.